.github/labeler.yml

```
@@ -4,3 +4,18 @@ documentation:
 explorer:
   - changed-files:
       - any-glob-to-any-file: apps/explorer/**/*
+wallet:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet/**/*
+wallet-dashboard:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet-dashboard/**/*
+ts-sdk:
+  - changed-files:
+      - any-glob-to-any-file: sdk/typescript/**/*
```

we should mark it as the TS SDK if it matches any of the packages inside
```suggestion
      - any-glob-to-any-file: sdk/**/*
```